File: GO/euler005.go
package main

import (
	"fmt"
)

func main() {
	fmt.Println(solve())
}

func solve() int {
	LIMIT := 20
	i := 1
	check := 0
	for true {
		i++
		check = 0
		for j := 1; j < LIMIT+1; j++ {
			if i%j == 0 {
				check++
			} else {
				break
			}
		}
		if check == LIMIT {
			return i
		}
	}
	return -1
}

File: C#/euler005.cs
using System;

namespace ProjectEuler {
    class euler005 {
        static void Main() {
            Console.WriteLine(Solve());
        }

        static int Solve() {
            for (int i = 1; ; i++) {
                if (divisble(i)) {
                    return i;
                }
            }
        }

        static bool divisble(int n) {
            int LIMIT = 20;
            for (int i = 1; i <= LIMIT; i++) {
                if (n%i != 0) {
                    return false;
                }
            }
            return true;
        }
    }
}